ios/Video/RCTVideo.h

```diff
@@ -14,7 +14,9 @@
 
 @class RCTEventDispatcher;
 #if __has_include(<react-native-video/RCTVideoCache.h>)
-@interface RCTVideo : UIView <RCTVideoPlayerViewControllerDelegate, DVAssetLoaderDelegatesDelegate>
+@interface RCTVideo : UIView <RCTVideoPlayerViewControllerDelegate, DVAssetLoaderDelegatesDelegate, AVAssetResourceLoaderDelegate>
+#elif TARGET_OS_TV
+@interface RCTVideo : UIView <RCTVideoPlayerViewControllerDelegate>
```

Comment:
Is the `DVAssetLoaderDelegatesDelegate` also incompatible with tvOS?

Reply:
Yes, it appears so since it compiles. Also made me realize I shouldn't have copied in `AVAssetResourceLoaderDelegate` when I started working. That delegate does not exist in `master`.